pkg/api/controllers/metrics.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
The OpenSDS Authors -- please update all.

Reply:
done